PowerShellTools.HostService/ServiceManagement/Debugging/PowerShellDebuggingService.cs

```diff
@@ -931,9 +1057,9 @@ public IEnumerable<CallStack> GetCallStack()
                             (string)psFrame.FunctionName.ToString(),
                             (int)psFrame.ScriptLineNumber));
                 }
-                else if(GetDebugScenario() == DebugScenario.LocalAttach)
+                else if(GetDebugScenario() == DebugScenario.RemoteAttach || GetDebugScenario() == DebugScenario.LocalAttach)

```

Comment:
??
I don't think the debugscenario is expected to change in the scope of setbreakpoint(), right?